designer/client/page-edit.js

```diff
@@ -25,10 +26,19 @@ export class PageEdit extends React.Component {
       section: page?.section ?? "",
       isEditingSection: false,
       errors: {},
+      toggle: false,
     };
     this.formEditSection = React.createRef();
   }
 
+  async componentDidMount() {
+    const result = await new ToggleApi().fetchToggles();
+
+    this.setState({
+      toggle: result["featureEditPageDuplicateButton"] == true,
+    });
+  }
+
```

Comment:
I wonder if this would be better suited to being in the DataContext? This fetch is going to happen every time this component mounts (and any other components which might use a toggle). The env variables aren't going to change when it's deployed, so fetching is wasteful in this case

Reply:
We can add another context - but you end up in a bit of a wrapping hell when you have to unit test